r/vignettes/install.Rmd

```diff
@@ -175,25 +181,24 @@ tune one of several parameters. Here are some known complications and ways to ad
 If you get an error like
 
 ```
-Cannot call io___MemoryMappedFile__Open(). Please use arrow::install_arrow() to install required runtime libraries.
+Cannot call io___MemoryMappedFile__Open(). See https://arrow.apache.org/docs/r/articles/install.html for help installing Arrow C++ libraries.
 ```
 
 for every `arrow` function you call,
 that means that installing the package failed to retrieve or build C++ libraries
-compatible with the current version of the R package. `install_arrow()` provides
-some convenience wrappers around the various environment variables described below
-and has defaults that should isolate the package installation and make it
-more likely to succeed. See its help page for details.
-
-As of version 0.17, it is expected that C++ dependencies should be built successfully
-on all Linux distributions, so if you see this message because the C++ libraries
-errored while building,
+compatible with the current version of the R package.
+
+It is expected that C++ dependencies should be built successfully
+on all Linux distributions, so you should not see this message. If you do,
+please check the "Known installation issues" below to see if any apply.
+If none apply, retry the installation with `arrow::install_arrow(verbose = TRUE)`
+so that details on what failed are shown, then
 please [report an issue](https://issues.apache.org/jira/projects/ARROW/issues).
```

Comment:
Do we want to proactively ask for the install log in the report? I've seen a few jiras where that's usually the first ask.

Reply:
Yeah the first ask is almost always to rerun with verbosity (what I added in this PR) and share the logs. Probably would be good to explicitly ask for the full output here.